uniconnect-epfl/uniconnect
# feat: announcements added to the Home screen

# What I did
I did a refont of the all HomeScreen, adding the annouccement Screen
I refined the styling for it to match the expected design from Figma, adding the add event button and the correct layout

# How I did it
I used the Section List component from React to have a smoother integration of the events on the page. 
Separated the HomeScreen with 2 subsections EventScreen and AnnoucementScreen
I modified the EventCard to add attributes useful for the project.
I created the Announcement component and the Announcement. 
I tested everything
# How to verify it
Go to the Home Screen and switch between tabs to explore the announcements and events.
You can also add an Event or an Announcement. The new Events need a refresh to appear on the list.

# Demo video


https://github.com/uniconnect-epfl/uniconnect/assets/108724823/485ce0d3-23e2-4a0b-a091-d8273cc50939




# Pre-merge checklist
The changes I have introduced:
